src/content/en/2022/performance.md

```diff
@@ -1,22 +1,989 @@
+
+### Interaction to Next Paint (INP)
+
+<a hreflang="en" href="https://web.dev/inp/">Interaction to Next Paint</a> (INP) measures the amount of time it takes for the browser to complete the next paint in response to a user interaction. This metric was created after Google <a hreflang="en" href="https://web.dev/responsiveness/">requested feedback</a> on a proposal to improve how we measure responsiveness. Many readers may be hearing about this metric for the first time, so it's worth going into a bit more detail about how it works.
+
+An _interaction_ in this context refers to the user experience of providing an input to a web application and waiting for the next frame of visual feedback to be painted on the screen. The only inputs that are considered for INP are clicks, taps, and key presses. The INP value itself is taken from one of the worst interaction latencies on the page. Refer to the <a hreflang="en" href="https://web.dev/inp/#what-is-inp">INP documentation</a> for more info on how it's calculated.
+
+Unlike FID, INP is a measure of all interactions on the page, not just the first one. It also measures the entire time until the next frame is painted, unlike FID which only measures the time until the event handler only starts_ _processing. In these ways, INP is a much more representative metric of the holistic user experience on the page.
+
+A website has "good" INP if 75% of its INP experiences are faster than 200 ms. A website has "poor" INP if the 75th percentile is greater than or equal to 500 ms. Otherwise, it's INP is assessed as "needs improvement".
+
+{{ figure_markup(
+  image="inp-device.png",
+  caption="Distribution of INP performance by device.",
+  description="Bar chart showing the percent of websites having good, needs improvement, or poor INP by device. 95% of desktop websites have good INP, 4% need improvement, and 1% are poor. 55% of mobile websites have good INP, 36% need improvement, and 8% are poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR-dJP3uphZoGE5A_luniNBFm5V2ww6irfOxANg0hrMid7gjgrtchsN_utOIDOvVZUjIwpmUBb27nHF/pubchart?oid=755106375&format=interactive",
+  sheets_gid="555510064",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+55% of websites have "good" INP on mobile, 36% are rated "needs improvement", and 8% have "poor" INP. The desktop story of INP is more similar to FID in that 95% of websites are rated "good", 4% are rated "needs improvement", and 1% are "poor".
+
+The enormous disparity between desktop and mobile users' INP experiences is much wider than with FID. This illustrates the extent to which mobile devices are struggling to keep up with the overwhelming amount of work websites do, and all signs point to the increasing reliance on <a hreflang="en" href="https://httparchive.org/reports/state-of-javascript?start=earliest&end=latest&view=grid#bytesJs">JavaScript</a> as a major factor.
```

Comment:
Prefer Web Almanac over HTTP Archive resources?

```suggestion
The enormous disparity between desktop and mobile users' INP experiences is much wider than with FID. This illustrates the extent to which mobile devices are struggling to keep up with the overwhelming amount of work websites do, and all signs point to the increasing reliance on [JavaScript](./javascript#how-much-javascript-do-we-load) as a major factor.
```

Reply:
I used the HA report in this case because it shows the change in JS over time

Reply:
Fair enough - as long as there's a reason. Resolving.